mars-sim-core/src/main/java/com/mars_sim/core/person/ai/task/util/Worker.java

```diff
@@ -145,27 +92,6 @@ public interface Worker extends UnitIdentifer, EquipmentOwner, SkillOwner {
 	 */
 	public void setMission(Mission newMission);
 
-	/**
-	 * Gets the Worker's position within the Settlement/Vehicle.
-	 * 
-	 * @return
-	 */
-	public LocalPosition getPosition();
-	
-	/**
-	 * Sets the worker's position at a settlement.
-	 *
-	 * @param position
-	 */
-	public void setPosition(LocalPosition position);
-
-	/**
-	 * Sets the building the worker is located at.
-	 *
-	 * @param position
-	 */
-	public void setCurrentBuilding(Building building);
```

Comment:
guess we no longer need this now for proactively setting its whereabout. 

Is it currently done by the activity spot related methods ?

Reply:
Good question, but will still need it. The point these methods are now moved to the Mobile unit interface. A Worker is also mobile which makes perfect sense